examples/compiled/interactive_seattle_weather.vg.json

```diff
@@ -562,14 +562,7 @@
       "stroke": "color",
       "gradientLength": {"signal": "clamp(concat_0_height, 64, 200)"},
       "symbolType": "circle",
-      "encode": {
-        "symbols": {
-          "update": {
-            "fill": {"value": "transparent"},
-            "opacity": {"value": 0.7}
-          }
-        }
-      },
+      "encode": {"symbols": {"update": {"opacity": {"value": 0.7}}}},
```

Comment:
@domoritz Aha point symbol needs the override.

Reply:
or is this correct? The bar chart layer triggers the fill. Should the legend have circles (to match bar chart) or points (to match scatterplot)?

[example](https://vega.github.io/editor/#/examples/vega-lite/interactive_seattle_weather)

Reply:
We have to decide. I don't remember what the default resolution is (first or last legend when merging). @kanitw?

Reply:
What do you mean override here?

Reply:
The `fill` attribute in `... encode.symbols.update` overriding the `fill` attribute at the top level `legend`.

Reply:
btw if we decide to remove this override, we probably want to make sure the opacity of the fill matches the opacity of the stroke.